MultiplayerMod/Platform/Steam/Network/SteamClient.cs

```diff
@@ -121,7 +121,7 @@ public class SteamClient : IMultiplayerClient {
         SteamFriends.SetRichPresence("connect", $"+connect_lobby {lobby.Id}");
     }
 
-    private SteamNetworkingIdentity GetNetworkingIdentity(CSteamID steamId) {
+    private static SteamNetworkingIdentity GetNetworkingIdentity(CSteamID steamId) {
```

Comment:
I propose not to use static in places where it's not supposed to be.

Reply:
Done